### PR TITLE
Hook into purge PD methods from purge script.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

**Comment by ashercodeorg** [comment]
@Hamms: Note the TODO that I'm assigning to you.

File: lib/cdo/delete_accounts_helper.rb [hunk 1]
```diff
@@ -97,12 +103,24 @@ def self.purge_orphaned_students(user_id)
...
+  # Remove all user generated content associated with any PD the user has been through, as well as
+  # all PII associated with any PD records.
   # @param [Integer] The ID of the user to clean the PD content.
   def self.clean_and_destroy_pd_content(user_id)
     PeerReview.where(reviewer_id: user_id).each(&:clear_data)
 
     Pd::TeacherApplication.where(user_id: user_id).each(&:destroy)
+    Pd::FacilitatorProgramRegistration.where(user_id: user_id).each(&:clear_form_data)
+    Pd::RegionalPartnerProgramRegistration.where(user_id: user_id).each(&:clear_form_data)
+    Pd::WorkshopMaterialOrder.where(user_id: user_id).each(&:clear_form_data)
```
Comment (aoby) [inline lib/cdo/delete_accounts_helper.rb]: This one is not a form. It's [clear_data](https://github.com/code-dot-org/code-dot-org/blob/5e545f1cbad96efce48bab1456a675d7bcd9b7c9/dashboard/app/models/pd/workshop_material_order.rb#L198)

File: lib/cdo/delete_accounts_helper.rb [hunk 2]
```diff
@@ -97,12 +103,24 @@ def self.purge_orphaned_students(user_id)
...
+
+    # TODO(hamms): This query is not indexed. Add an index so that it is.
+    pd_enrollment_id = Pd::Enrollment.where(user_id: user_id).pluck(:id).first
+    if pd_enrollment_id
+      Pd::TeacherconSurvey.where(pd_enrollment_id: pd_enrollment_id).each(&:clear_form_data)
+      Pd::WorkshopSurvey.where(pd_enrollment_id: pd_enrollment_id).each(&:clear_form_data)
+      Pd::Enrollment.where(id: pd_enrollment_id).each(&:clear_form_data)
```
Comment (aoby) [inline lib/cdo/delete_accounts_helper.rb]: And here: [clear_data](https://github.com/code-dot-org/code-dot-org/blob/b1ef632a91952668e67ef62badf80a97505b84aa/dashboard/app/models/pd/enrollment.rb#L246)

Comment (ashercodeorg) [reply lib/cdo/delete_accounts_helper.rb]: Done. Sorry for sloppiness.

File: lib/cdo/delete_accounts_helper.rb (hunk 1 above)
Comment (ashercodeorg) [reply lib/cdo/delete_accounts_helper.rb]: Done. Sorry for sloppiness.

Comment (aoby) [reply lib/cdo/delete_accounts_helper.rb]: No worries! That's what reviews are for. You've certainly caught plenty similar errors of mine :)

**Pull request merged**